cpp/src/parquet/reader_test.cc

```diff
@@ -322,6 +323,27 @@ TEST(TestFileReaderAdHoc, NationDictTruncatedDataPage) {
   ASSERT_EQ(ss2.str(), ss.str());
 }
 
+TEST(TestFileReaderEncoding, DeltaBinaryPacked) {
+  // Parquet file(delta_binary_packed.parquet) is generated by parquet-mr version 1.10.0.
+  // There are 66 columns in total and their encoding type is DELTA_BINARY_PACKED. The
+  // data type of the first 65 columns is bigint and their bit width ranges from 0 to 64.
+  // The data type of the last column is int.
+  std::list<int> columns;
+
+  std::stringstream ss_values;
+  const char* file = "delta_binary_packed.parquet";
+  auto reader_props = default_reader_properties();
+  auto reader = ParquetFileReader::OpenFile(data_file(file), false, reader_props);
+  ParquetFilePrinter printer(reader.get());
+  printer.DebugPrint(ss_values, columns, true, false, false, file);
+
+  std::ifstream in(data_file("delta_binary_packed.parquet.expect"));
```

Comment:
this test seems very brittle.  using CSV or JSON or some other format and comparing the column data seems directly instead of the string representation seems like a better idea?